src/Blocks/TableOfContents.php

```diff
@@ -47,7 +48,7 @@ public function register_submenu_block(): void
                         'type' => 'string',
                         'default' => '',
                     ],
-                    'submenu_style' => [ // Needed for old blocks conversion.
+                    'table_of_contents_style' => [ // Needed for old blocks conversion.
```

Comment:
Same comment as earlier, should remain `submenu_style`

Reply:
Change reverted.